src/integrations/ethereum/renderer/screens/contracts/ContractDetails.js

```diff
@@ -75,29 +75,29 @@ class ContractDetails extends Component {
     const title = encodeURIComponent(
       `Decoding Error when running Ganache ${app.getVersion()} on ${
         process.platform
-      }`,
+      }`
     );
 
     const body = this.renderIssueBody(
-      sanitizeError(this.props.workspaces.current.shownContract.state.error),
+      sanitizeError(this.props.workspaces.current.shownContract.state.error)
     );
 
     shell.openExternal(
-      `https://github.com/trufflesuite/ganache/issues/new?title=${title}&body=${body}`,
+      `https://github.com/trufflesuite/ganache-ui/issues/new?title=${title}&body=${body}`
```

Comment:
This is the only actual change in this file (updating the link to point to the correct project)